### ISSUE TITLE
Tests fail with Stenciller 0.1400

### ISSUE THREAD

**Issue by eserte** [Csson/p5-Dist-Zilla-Plugin-Stenciller-HtmlExamples#2]
Tests started to fail on my smoker systems:

```
#   Failed test 'Dist/Zilla/Plugin/Stenciller/HtmlExamples.pm loaded ok'
#   at t/00-compile.t line 37.
#          got: '65280'
#     expected: '0'
syntax error at /tmpfs/.cpan-build-cpansand/2016021618/Dist-Zilla-Plugin-Stenciller-HtmlExamples-0.0200-AC8cCf/blib/lib/Dist/Zilla/Plugin/Stenciller/HtmlExamples.pm line 60, near "Maybe["
Bareword "Dir" not allowed while "strict subs" in use at /tmpfs/.cpan-build-cpansand/2016021618/Dist-Zilla-Plugin-Stenciller-HtmlExamples-0.0200-AC8cCf/blib/lib/Dist/Zilla/Plugin/Stenciller/HtmlExamples.pm line 33.
Bareword "Str" not allowed while "strict subs" in use at /tmpfs/.cpan-build-cpansand/2016021618/Dist-Zilla-Plugin-Stenciller-HtmlExamples-0.0200-AC8cCf/blib/lib/Dist/Zilla/Plugin/Stenciller/HtmlExamples.pm line 42.
Bareword "Path" not allowed while "strict subs" in use at /tmpfs/.cpan-build-cpansand/2016021618/Dist-Zilla-Plugin-Stenciller-HtmlExamples-0.0200-AC8cCf/blib/lib/Dist/Zilla/Plugin/Stenciller/HtmlExamples.pm line 45.
Bareword "AbsFile" not allowed while "strict subs" in use at /tmpfs/.cpan-build-cpansand/2016021618/Dist-Zilla-Plugin-Stenciller-HtmlExamples-0.0200-AC8cCf/blib/lib/Dist/Zilla/Plugin/Stenciller/HtmlExamples.pm line 54.
Bareword "Bool" not allowed while "strict subs" in use at /tmpfs/.cpan-build-cpansand/2016021618/Dist-Zilla-Plugin-Stenciller-HtmlExamples-0.0200-AC8cCf/blib/lib/Dist/Zilla/Plugin/Stenciller/HtmlExamples.pm line 58.
Compilation failed in require at -e line 1.
# Looks like you failed 1 test of 1.
t/00-compile.t ......... 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/1 subtests 
```

Statistical analysis suggests that the problem is caused by Stenciller 0.1400:

```
****************************************************************
Regression 'mod:Stenciller'
****************************************************************
Name                   Theta          StdErr     T-stat
[0='const']           1.0000          0.0000    92989851403350656.00
[1='eq_0.1400']      -1.0000          0.0000    -48393452990440776.00

R^2= 1.000, N= 48, K= 2
****************************************************************
```

**Comment by Csson** [comment]
Hi,

Thanks for the report, I'll look into what might be the cause.

**Comment by Csson** [comment]
Fixed in 0.0201.
